packages/core/src/components/meter/meter.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.hasCenterTitle {
+    justify-content: center;
+  }
+
+  &.hasMarginTop {
+    margin-bottom: 0;
+    margin-top: var(--spacing-xsmall);
+  }
+
+  &.is-mobile {
+    font: var(--text-body-small);
+    font-weight: 300;
+    letter-spacing: var(--text-body-small-letter);
+  }
+
+  * {
+    margin: 0;
+  }
+}
+
+.title {
+  font: var(--text-body-medium);
+  font-weight: 300;
+  letter-spacing: var(--text-body-medium-letter);
+
+  &.is-mobile {
+    font: var(--text-body-small);
+    font-weight: 300;
+    letter-spacing: var(--text-body-small-letter);
+  }
+}
+
+.atom-meter {
+  background-color: var(--color-neutral-light-4);
+  border-radius: var(--border-radius-medium);
+  height: var(--spacing-base);
+  position: relative;
+  width: 100%;
+
+  &.is-small {
+    height: var(--spacing-xsmall);
+
+    .within {
```

Comment:
question: what is `within`, it is really a good name for it?

Reply:
uhmmm, maybe `step`?

Reply:
I think it makes sense, like the current step in the progress bar

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/7a1570d82141d9ce0aafd4dd4efccea5fce865c9